AudioBookConverter won't work with certain MP3 files

I have used AudioBookConverter several times, so I have had it working to create .M4B files before.
Today I after converting 1 book successfully, I tried to convert a second one and the conversion didn;t work.
After starting the conversion, every input (qued MP3 files, title, author name etc.) are **cleared** as if the program was restarted.

Looking at the log file it seems, an exception was triggered due to the error: **Cannot read field "title" because "chapter.tags"** 

Is this bug ?
Or is ut my MP3 file ?
If the latter, is there a way to fix my MP3 file to circumvent this error ?



`28-08-2023 00:55:56 ERROR Book:382 - Error constructing book:
java.lang.RuntimeException: java.util.concurrent.ExecutionException: java.lang.NullPointerException: Cannot read field "title" because "chapter.tags" is null
	at uk.yermak.audiobookconverter.loaders.MediaInfoProxy.getMediaInfo(MediaInfoProxy.java:20)
	at uk.yermak.audiobookconverter.loaders.MediaInfoProxy.getBookInfo(MediaInfoProxy.java:80)
	at uk.yermak.audiobookconverter.book.Book.construct(Book.java:33)
	at uk.yermak.audiobookconverter.ConversionGroup.launch(ConversionGroup.java:130)
	at uk.yermak.audiobookconverter.fx.FilesController.lambda$start$15(FilesController.java:296)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
Caused by: java.util.concurrent.ExecutionException: java.lang.NullPointerException: Cannot read field "title" because "chapter.tags" is null
	at java.base/java.util.concurrent.FutureTask.report(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.get(Unknown Source)
	at uk.yermak.audiobookconverter.loaders.MediaInfoProxy.getMediaInfo(MediaInfoProxy.java:18)
	... 9 more
Caused by: java.lang.NullPointerException: Cannot read field "title" because "chapter.tags" is null
	at uk.yermak.audiobookconverter.loaders.MediaInfoLoader.processEmbededChapter(MediaInfoLoader.java:124)
	at uk.yermak.audiobookconverter.loaders.MediaInfoLoader.call(MediaInfoLoader.java:103)
	at uk.yermak.audiobookconverter.loaders.MediaInfoLoader.call(MediaInfoLoader.java:24)
	... 4 more`

--- a/src/main/java/uk/yermak/audiobookconverter/loaders/MediaInfoLoader.java
+++ b/src/main/java/uk/yermak/audiobookconverter/loaders/MediaInfoLoader.java
@@ -120,8 +120,13 @@ class MediaInfoLoader implements Callable<MediaInfo> {
         AudioBookInfo bookInfo = mediaInfo.getBookInfo();
         for (int i = 0; i < chapters.size(); i++) {
             FFmpegChapter chapter = chapters.get(i);
-            Track track = new Track(StringUtils.leftPad(String.valueOf(i + 1), 3, "00"));
-            track.setTitle(chapter.tags.title);
+            String trackNo = StringUtils.leftPad(String.valueOf(i + 1), 3, "00");
+            Track track = new Track(trackNo);
+            if (chapter.tags != null) {
+                track.setTitle(chapter.tags.title);
+            } else {
+                track.setTitle("Chapter " + trackNo);
+            }
             track.setStart((long) (Double.parseDouble(chapter.start_time) * 1000));
             track.setEnd((long) (Double.parseDouble(chapter.end_time) * 1000));
             bookInfo.tracks().add(track);

